### PR TITLE
Build was failing on Linux with OpenJDK. Fixed instructions.

### PR DESCRIPTION
**Pull request by nicolas-raoul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,22 @@ __[:arrow_down: Download](https://github.com/yarl/pattypan/releases)__
...
+- [Apache Ant](https://ant.apache.org/)
+- the JavaFX libraries (included in Oracle's JDK but not in OpenJDK)
+
+Then run these commands in a terminal:
 
 ```
 git clone https://github.com/yarl/pattypan.git
 cd pattypan
-ant package-for-store
+ant javafx.deploy.includeDT=true package-for-store
```
Comment (samwilson) [inline README.md]: Missing `-D`

File: README.md [hunk 2]
```diff
@@ -6,13 +6,22 @@ __[:arrow_down: Download](https://github.com/yarl/pattypan/releases)__
...
+This program is being developed using [NetBeans IDE](https://netbeans.org/).
+
 ### Build and run
-Program is being written using [NetBeans IDE](https://netbeans.org/) and [Apache Ant](https://ant.apache.org/) is used for building. In order to download and build source code, do following:
+In order to build Pattypan, please:
+
+Download and install:
+
+- [Apache Ant](https://ant.apache.org/)
+- the JavaFX libraries (included in Oracle's JDK but not in OpenJDK)
```
Comment (samwilson) [inline README.md]: Add a comment about `sudo apt install openjfx`?

**Comment by samwilson** [comment]
I've just realised that this stuff is also documented on https://github.com/yarl/pattypan/wiki/Run#ubuntu

File: README.md (hunk 2 above)
Comment (Abbe98) [reply README.md]: And `sudo apt install ant`?

**Comment by Abbe98** [comment]
Would be great to get this updated and merged.

File: README.md (hunk 1 above)
Comment (michal-josef-spacek) [reply README.md:24]: That isn't enough for me:

Comment (michal-josef-spacek) [reply README.md:24]: With -D isn't working too.
> -do-compile:
>    [javac] Compiling 26 source files to /home/skim/44_git/pattypan/build/classes
>    [javac] warning: [options] bootstrap class path not set in conjunction with -source 8
>    [javac] /home/skim/44_git/pattypan/src/pattypan/Main.java:27: error: package javafx.application does not exist
>    [javac] import javafx.application.Application;
>    [javac]                          ^
> ...

> `skim@woman:~/44_git/pattypan$ java -version`
> openjdk version "11.0.9.1" 2020-11-04
> OpenJDK Runtime Environment (build 11.0.9.1+1-post-Debian-1deb10u2)
> OpenJDK 64-Bit Server VM (build 11.0.9.1+1-post-Debian-1deb10u2, mixed mode, sharing)

**Comment by Abbe98** [comment]
Closing as Pattypan has been migrated to Java 11+.

**Pull request closed without merging**